README.md

```diff
@@ -1,5 +1,25 @@
-# DEV1-IMU
+# DEV1 IMU
 
 ## Introduction
 
-*The Inertial Measurement Unit senses the acceleration of the bike and reports any relevant data over CAN.*
\ No newline at end of file
+### DEV1 IMU
+
+The Inertial Measurement Unit measures linear and angular acceleration of the bike using the BNO055 chip. More metrics (gravity, linear and angular velocity, pitch/yaw/roll, and the surrounding magnetic field) can be calculated from the two. The IMU also reports any relevant data over CAN. Such data provides general information to the bike on its orientation and allows other boards to act accordingly for optimal performance. The system is based around the built-in BNO055 chip. The firmware is written for the STM32F334K8U6 microcontroller designed into the DEV1 IMU.
+
+The DEV1 IMU will have the following responsibilities:
+1. Measure linear and angular acceleration in 3D space (X,Y,Z) of bike
+2. Expose the system over the DEV1 CAN network
```

Comment:
```suggestion
2. Report measurements over the DEV1 CAN network
```
The reason for this difference from the BMS is that the BMS can be controlled over CAN, whereas the IMU does not take any inputs from CAN